src/components/network-map-viewer/network/draw-control.ts

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import MapboxDraw from '@mapbox/mapbox-gl-draw';
+import { UUID } from 'crypto';
+import { useCallback } from 'react';
+import { useControl } from 'react-map-gl';
+
+import type { ControlPosition } from 'react-map-gl';
+
+// eslint-disable-next-line no-var
+var MapDrawerController: MapboxDraw | undefined = undefined;
+
+export function getMapDrawer() {
+    // Add your custom logic here
+    return MapDrawerController;
+}
```

Comment:
This function sems unused

Reply:
this one is used in gridstudy , the function is exposed in network-map.jsx
```
useImperativeHandle(ref, () => ({
       ...
        getSelectedLines,
        **getMapDrawer,**
    }));
```
i deleted the comment